common/actions/initialize/main.js

```diff
@@ -1,16 +1,18 @@
-const utils = require('../../../utils');
+const { getInput, execStep, setFailed } = require('../../../utils');
 
 try{
-    const ghToken = utils.getInput('gh-token');
+    const npmNexusAuth = getInput('nexus-token');
```

Comment:
This migration requires all pipelines using `nexus-token` (and its corresponding SECRET setup under Github Marfeel's org)

Reply:
Depending on this https://github.com/Marfeel/github-actions/pull/28/files#r499600498, this comments will become outdated